Sources/web3swift/Web3/Web3+Options.swift

```diff
@@ -159,11 +158,11 @@ public struct TransactionOptions {
         return newOptions
     }
 //
-//    /// merges two sets of options along with a gas estimate to try to guess the final gas limit value required by user.
```

Comment:
Deletes DocC string comment (///) Again appears only in this file.

Reply:
commented-out doc comment,so no impact. This code is dead anyway, which is why I didn't restore it.